### PR TITLE
Add simple FCT unit tests

### LINKED ISSUE #871: Add simple FCT unit tests
**Is your feature request related to a problem? Please describe.**
#856 added the simple FCT as an integration test directly. But it's better to add unit tests as well.

**Describe the solution you'd like**
Add dedicated unit tests, by replicating the existing third-order upwinding operator unit tests and verify that indeed we fall back on the existing behavior.

### PR DESCRIPTION
**Pull request by valeriabarra**
This will close #871 

- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: test/Operators/finitedifference/column.jl [hunk 1]
```diff
@@ -775,6 +775,196 @@ end
...
+    @test conv_adv_wc[3] ≈ 3 atol = 0.1
+    @test conv_adv_wc[1] ≤ conv_adv_wc[2] ≤ conv_adv_wc[2]
+end
+
+@testset "Simple FCT: lin combination of UpwindBiasedProductC2F + Upwind3rdOrderBiasedProductC2F on (uniform and stretched) non-periodic mesh, with FirstOrderOneSided BCs" begin
+    FT = Float64
+    n_elems_seq = 2 .^ (4, 6, 8, 10)
+    stretch_fns = (Meshes.Uniform(), Meshes.ExponentialStretching(1.0))
+
+    for (i, stretch_fn) in enumerate(stretch_fns)
```
Comment (charleskawczynski) [inline test/Operators/finitedifference/column.jl:839]: It's probably easier to write this for only the exponential case because the convergence orders won't match up in general

File: test/Operators/finitedifference/column.jl [hunk 2]
```diff
@@ -775,6 +775,196 @@ end
...
+                top = Operators.SetValue(
+                    Geometry.Contravariant3Vector(FT(0.0)),
+                ),
+            )
+
+            corrected_antidiff_flux = @. divf2c(
+                C * (third_order_fluxᶠ(w, c) - first_order_fluxᶠ(w, c)),
+            )
+            adv_wc =
+                @. @. divf2c.(first_order_fluxᶠ(w, c)) + corrected_antidiff_flux
```
Comment (charleskawczynski) [inline test/Operators/finitedifference/column.jl:886]:
```suggestion
            adv_wc =
                @. divf2c(first_order_fluxᶠ(w, c)) + corrected_antidiff_flux
```

Comment (valeriabarra) [reply test/Operators/finitedifference/column.jl:886]: Good eye! Sorry that slipped by accident

File: test/Operators/finitedifference/column.jl [hunk 3]
```diff
@@ -775,6 +775,196 @@ end
     end
 end
 
+
+@testset "Simple FCT: lin combination of UpwindBiasedProductC2F + Upwind3rdOrderBiasedProductC2F on (uniform) periodic mesh" begin
```
[the remark this answers is not in the document]
Comment (valeriabarra) [reply test/Operators/finitedifference/column.jl:779]: I think these are already covered in the integration tests (some of which use quantitative tests to check against expected behavior). However, it would be good to improve some of our simple unit tests as well. But I don't think this PR is the right venue for that. We should discuss perhaps in a separate issue how to improve our unit tests. These are not the only one which need some improvement and/or corner cases covered.

File: test/Operators/finitedifference/column.jl (hunk 1 above)
Comment (valeriabarra) [reply test/Operators/finitedifference/column.jl:839]: It was very close and some cases also better actually. I don't think it would be the right venue here to split them up and reformulate these tests

**Comment by valeriabarra** [comment]
bors try

**Review by charleskawczynski — APPROVED** [review]

**Comment by valeriabarra** [comment]
@charleskawczynski , I am not familiar with the error in the CI job:
```
🚨 Error: The command was interrupted by a signal
```
Do we just punch it again?

**Comment by valeriabarra** [comment]
Might be related to time-out. Should we increase it?
